master/internal/config/config.go

```diff
@@ -250,6 +252,13 @@ func (c *Config) Resolve() error {
 		return err
 	}
 
+	// CHECK: how do we generally check for and report deprecated config options?
```

Comment:
I would vote for removing it all together, it was prefixed with an underscore and undocumented in master config page, really doubt anyone used it

Reply:
Can we confirm no one is using it? If so, maybe then it's good to rip the bandaid off, but I think the safe thing to do is do the check for the option, let them know the option is being deprecated, and then remove it next major release.

Reply:
if someone does use it and we take this out master won't come up.

Reply:
I'll land this as is if @eecsliu approves as well since I want to update other prs/branches depending on this sooner but I'd be happy to take the rest out once we wrap up this conversation.

Reply:
chatted w/ @eecsliu offline.